### DISCUSSION TITLE
Endless mode CPU repair (recover lost "lives")

### DISCUSSION THREAD

**Discussion by The-JMFB** [ochadenas/cpudefense#149] [Ideas]
I think one of the endless mode only features should be the ability to tap on the CPU and purchase a lost "life". The thinking here isn't to allow a player to simply purchase their way to victory, but rather a way to salvage a late stage victory after a swarm or perhaps a slightly weak defense layout. The price should be prohibitive, perhaps starting at 60 B and increasing with each purchase. This would allow someone an opportunity to not lose with just a few remaining waves because something like a swarm or a mem chip cooldown let an attacker by. 

I find that by wave 75 or so, I'm not spending inf anymore and the only thing left is to occasionally purge my mem chips and watch. Nothing left to do to bolster my defense so I just have to take what I get from the swarms. Being able to purchase that recovery would have been great on more than a few occasions... especially those that I stupidly used fast forward in late waves (don't do it! You'll have a noticeably higher number of lost lives using fast forward, no matter how many mem slots are available).

**Comment by The-JMFB** [comment]
Another thought on this topic: at higher endless levels, I am frequently collecting 10 or 15 crypto coins. As these currently have no other value for me anyway, perhaps an increasing number of coins would be a better purchase price for this feature. The first recovered life is 5 coins, the next 7, the next 10 and so on.

**Comment by ochadenas** [comment]
Added this in 1.43 as a feature.

I'm not sure about the pricing. Currently, it is 1 coin for the first life recovered, 2 for the second, and so on. Per attempt, i.e. if you replay the level, the price starts again at 1 coin. For each 32 levels, prices increase by 1 coin.

Do you think that this is too cheap?
